bindings/python/bt2/bt2/notification.py

```diff
@@ -210,7 +212,7 @@ def __init__(self, stream):
     def stream(self):
         stream_ptr = native_bt.notification_stream_begin_get_stream(self._ptr)
         assert(stream_ptr)
-        return bt2.stream._create_from_ptr(stream_ptr)
+        return bt2._Stream._create_from_ptr(stream_ptr)
```

Comment:
Not sure about this. Why not using the IR domain's `create_stream_from_ptr()`?

Reply:
done.